apps/src/componentLibrary/alert/alert.module.scss

```diff
@@ -19,50 +19,50 @@ $alert-span-vertical-top-spacing: 2px;
     }
 
     a {
-      color: $light_black;
+      color: var(--text-neutral-primary);
     }
   }
 }
 
 // Types (colors)
 .alert-primary {
-  background-color: $light_secondary_100;
+  background-color: var(--background-brand-purple-light);
 }
 
 .alert-success {
-  background-color: $light_affirmative_100;
+  background-color: var(--background-success-light);
 
   .alertContentContainer i {
-    color: $light_affirmative_500;
+    color: var(--text-success-primary);
   }
 }
 
 .alert-danger {
-  background-color: $light_negative_100;
+  background-color: var(--sentiment-error-10);
 
   .alertContentContainer i {
-    color: $light_negative_500;
+    color: var(--text-error-primary);
```

Comment:
Example of the new color variable in the inspector:
<img width="1377" alt="Screenshot 2024-11-27 at 9 52 18 AM" src="https://github.com/user-attachments/assets/4b0981d1-1a56-4f5b-aa00-ae56b1aaa53a">